src/Pos/Wallet/Web/Server/Methods.hs

```diff
@@ -391,16 +390,23 @@ getWAddressBalance :: WalletWebMode m => CWAddressMeta -> m Coin
 getWAddressBalance addr =
     getBalance <=< decodeCIdOrFail $ cwamId addr
 
+
```

Comment:
Ah yeah, that was me trying to coax github into updating the PR (github was having issues yesterday).